apps/remote_control/lib/lexical/remote_control/plugin/discovery.ex

```diff
@@ -0,0 +1,64 @@
+defmodule Lexical.RemoteControl.Plugin.Discovery do
+  @moduledoc """
+  Discovers any plugins in any loaded applications
+
+  This module runs through the loaded applications and checks to see if any of them are lexical plugins.
+  If a plugin is found, the app is then registered with the plugin system.
+
+  If we're running in a namespaced build, we must apply the same namespace to the plugin's modules, or it
+  will be looking for structs like `Lexical.Document', and be passed in structs like `LXRelease.Document`,
+  and the plugin will crash.
+  """
+  alias Lexical.Plugin
+  alias Mix.Tasks.Namespace
+
+  require Logger
+
+  def run do
+    for {app_name, _, _} <- :application.loaded_applications(),
```

Comment:
There's a bug here; I found that the `run/0` only works when you first open a project(I mean, `.lexical` does not exist).

Reply:
This can't be true; I've run this repeatedly with a project where `.lexical` exists.